src/includes/class-wp-cli.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Network Management Command
+ *
+ */
+class WPMultiNetwork_Command extends WP_CLI_Command {
+
+	/**
+	 * @var array $obj_fields Default fields to display for each object.
```

Comment:
Sorry, to clarify, I meant the `$obj_fields` in the docblock. It can just be `@var array Default ...` instead, as it's documenting a property directly. :)